diff -up k3b-20.03.80/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp.2~ k3b-20.03.80/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp
--- k3b-20.03.80/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp.2~	2020-03-24 01:40:25.729405475 +0100
+++ k3b-20.03.80/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp	2020-03-24 01:41:48.951426948 +0100
@@ -17,6 +17,16 @@
 
 #include <config-k3b.h>
 
+#define HAVE_FFMPEG_AVFORMAT_OPEN_INPUT
+#define HAVE_FFMPEG_AV_DUMP_FORMAT
+#define HAVE_FFMPEG_AVFORMAT_FIND_STREAM_INFO
+#define HAVE_FFMPEG_AVFORMAT_CLOSE_INPUT
+#define HAVE_FFMPEG_AVCODEC_OPEN2
+#define HAVE_FFMPEG_AVMEDIA_TYPE
+#define HAVE_FFMPEG_CODEC_MP3
+#define HAVE_FFMPEG_AVCODEC_DECODE_AUDIO4
+#define NEWFFMPEGAVCODECPATH
+
 extern "C" {
 /*
  Recent versions of FFmpeg uses C99 constant macros which are not present in C++ standard.
