--- k3b/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp.orig	2014-09-29 09:38:59.251811878 +0100
+++ k3b/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp	2014-09-29 10:06:13.082068499 +0100
@@ -44,29 +44,6 @@
 #define FFMPEG_CODEC(s) (s->codec)
 #endif
 
-#ifndef HAVE_FFMPEG_AVFORMAT_OPEN_INPUT
-//      this works because the parameters/options are not used
-#  define avformat_open_input(c,s,f,o) av_open_input_file(c,s,f,0,o)
-#endif
-#ifndef HAVE_FFMPEG_AV_DUMP_FORMAT
-#  define av_dump_format(c,x,f,y) dump_format(c,x,f,y)
-#endif
-#ifndef HAVE_FFMPEG_AVFORMAT_FIND_STREAM_INFO
-#  define avformat_find_stream_info(c,o) av_find_stream_info(c)
-#endif
-#ifndef HAVE_FFMPEG_AVFORMAT_CLOSE_INPUT
-#  define avformat_close_input(c) av_close_input_file(*c)
-#endif
-#ifndef HAVE_FFMPEG_AVCODEC_OPEN2
-#  define avcodec_open2(a,c,o) avcodec_open(a,c)
-#endif
-#ifndef HAVE_FFMPEG_AVMEDIA_TYPE
-#  define AVMEDIA_TYPE_AUDIO CODEC_TYPE_AUDIO
-#endif
-#ifndef HAVE_FFMPEG_CODEC_MP3
-#  define CODEC_ID_MP3 CODEC_ID_MP3LAME
-#endif
-
 // Has been deprecated for ages. Now removed. For now do what ffmpeg does
 // http://ffmpeg.org/pipermail/ffmpeg-cvslog/2012-August/053785.html
 // 192000 = 1 second of 48khz 32bit audio
@@ -329,24 +306,11 @@
         d->outputBufferPos = d->alignedOutputBuffer;
         d->outputBufferSize = AVCODEC_MAX_AUDIO_FRAME_SIZE;
 
-#ifdef HAVE_FFMPEG_AVCODEC_DECODE_AUDIO3
         int len = ::avcodec_decode_audio3(
-#else
-#  ifdef HAVE_FFMPEG_AVCODEC_DECODE_AUDIO2
-        int len = ::avcodec_decode_audio2(
-#  else
-        int len = ::avcodec_decode_audio(
-#  endif
-#endif
-
             FFMPEG_CODEC(d->formatContext->streams[0]),
             (short*)d->alignedOutputBuffer,
             &d->outputBufferSize,
-#ifdef HAVE_FFMPEG_AVCODEC_DECODE_AUDIO3
             &d->packet );
-#else
-            d->packetData, d->packetSize );
-#endif
 
         if( d->packetSize <= 0 || len < 0 )
             ::av_free_packet( &d->packet );
