--- k3b/libk3b/tools/k3blibdvdcss.cpp.orig	2014-05-11 21:45:22.591623077 +0100
+++ k3b/libk3b/tools/k3blibdvdcss.cpp	2014-05-11 21:46:09.109624959 +0100
@@ -273,7 +273,7 @@
 K3b::LibDvdCss* K3b::LibDvdCss::create()
 {
     if( !s_libDvdCss->isLoaded() ) {
-        s_libDvdCss->setFileName( "dvdcss" );
+        s_libDvdCss->setFileName( "dvdcss.so.2" );
         s_libDvdCss->setLoadHints( QLibrary::ExportExternalSymbolsHint );
 
         if( s_libDvdCss->load() ) {
